environment/src/main/java/com/sequenceiq/environment/environment/flow/creation/handler/NetworkCreationHandler.java

```diff
@@ -85,10 +85,10 @@ private boolean hasNetwork(Environment environment) {
                 && enabledPlatforms.contains(environment.getCloudPlatform());
     }
 
-    private void stepToFreeIpaCreation(Event<EnvironmentDto> environmentDtoEvent, EnvironmentDto environmentDto) {
+    private void initiateNextStep(Event<EnvironmentDto> environmentDtoEvent, EnvironmentDto environmentDto) {
```

Comment:
I prefer adding the name of the next step to this method. Flow is already complicated, help each other out with as much handrail as possible.